source/standards/configuration-management.html.md.erb

```diff
@@ -0,0 +1,24 @@
+---
+title: Configuration Management
+expires: 2018-05-16
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+When running VMs, such as in Amazon EC2, where the complexity of the configuration
+in the instance becomes too great for managing during deployment or within AMIs,
+use configuration management.
```

Comment:
Maybe not a statement for the first iteration of this guidance because I suspect it is somewhat controversial but I would like to have a statement that the consideration should be given to ensuring that where possible responsibilities of configuration management should be clearly delineated against the responsibilities of any application deployed on the instance. Close coupling of application behaviour and configuration management systems can lead to a complex service implementation which is difficult to evolve over time.

Reply:
Yeah, that's a really good point. I agree that's probably overkill for the first pass guidance too.